sharing_portal/templates/sharing_portal/list_day_pass_requests.html

```diff
@@ -0,0 +1,42 @@
+{% extends 'layout/default.html' %}
+{% load i18n cms_tags static pipeline bootstrap3 %}
+{% load render_bundle from webpack_loader %}
+
+{% block title %}Review Day Pass | Chameleon Cloud{% endblock %}
+
+{% block styles %}
+{{ block.super }}
+<link rel="stylesheet" type="text/css" href="{% static 'sharing_portal/day_pass.css' %}">
+{% endblock %}
+
+{% block content %}
+
+<div id="app">
+  <h1>Pending requests</h1>
+  {% for day_pass_request in pending_requests %}
+    <div class="day-pass-request">
+      <dt>Artifact</dt><dd>{{ day_pass_request.artifact.title }}</dd>
+      <dt>Name</dt><dd>{{ day_pass_request.name }}</dd>
+      <dt>Email</dt><dd>{{ day_pass_request.created_by.email }}</dd>
+      <dt>Institution</dt><dd>{{ day_pass_request.institution }}</dd>
+      <dt>Requested at</dt><dd>{{ day_pass_request.created_at }}</dd>
+      <a class="btn btn-primary" href="{{ day_pass_request.url }}" target="_blank">Review</a>
+    </div>
+  {% endfor %}
+  <h1>Reviewed requests</h1>
+  {% for day_pass_request in reviewed_requests %}
+    <div class="day-pass-request">
+      <dt>Artifact</dt><dd>{{ day_pass_request.artifact.title }}</dd>
+      <dt>Name</dt><dd>{{ day_pass_request.name }}</dd>
+      <dt>Email</dt><dd>{{ day_pass_request.created_by.email }}</dd>
+      <dt>Institution</dt><dd>{{ day_pass_request.institution }}</dd>
+      <dt>Reviewed by</dt><dd>{{ day_pass_request.decision_by }}</dd>
+      <dt>Reviewed at</dt><dd>{{ day_pass_request.decision_at }}</dd>
+      <dt>Status</dt><dd>{{ day_pass_request.status }}</dd>
+    </div>
+  {% endfor %}
+</div>
+
+<script id="artifact-data" type="application/json">{{ artifact_json|safe }}</script>
```

Comment:
Is this needed here? I think this was only put on some other templates because JS on the page needed access to the model data.